packages/web/src/scss/components/UNSTABLE_Footer/index.html

```diff
@@ -0,0 +1,1026 @@
+{{#> web/layout/plain }}
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Basic Usage</h2>
+  <div class="docs-Stack">
+
+    <footer class="UNSTABLE_Footer">
+      <div class="Container">
+        <div class="Grid Grid--cols-2 Grid--tablet--cols-4">
+          <nav aria-labelledby="footer-part-one" class="mb-600 mb-tablet-800">
+            <h3 class="typography-body-large-text-bold mb-600" id="footer-part-one">
+              Section headline
+            </h3>
+
+            <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+              <li>
+                <a href="www.example.com">
```

Comment:
can you please fix all example.com links to include https? Otherwise, it acts weird.

Reply:
WIll solve in the #1546